ambient.d.ts

```diff
@@ -19,22 +19,27 @@
  - Name Surname <name.surname@mojaloop.io>
 
  * Google
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  - Steven Wijaya <stevenwjy@google.com>
  --------------
  ******/
 
-import { Client } from '~/shared/ml-thirdparty-client'
+type Client = import('~/shared/ml-thirdparty-client').Client
+type Server = import('@hapi/hapi').Server
+type ServerApplicationState = import('@hapi/hapi').ServerApplicationState
 
-declare module '@hapi/hapi' {
-  interface ServerApplicationState {
-    mojaloopClient: Client
-  }
+interface NewServerApplicationState extends ServerApplicationState {
```

Comment:
I think it would be better to name it `LocalServerApplicationState` or `CustomServerApplicationState`

Reply:
`MojaServerApplicationState`

Reply:
or `PISPServerApplicationSteate`

Reply:
that makes more sense yes